docs/concepts/storage/persistent-volumes.md

```diff
@@ -118,8 +118,12 @@ For volume plugins that support the Delete reclaim policy, deletion removes both
 
 ### Expanding Persistent Volumes Claims
 
-With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. The current Alpha support was designed to only support volume types
-that don't need file system resizing (Currently only glusterfs).
+With Kubernetes 1.8, we have added Alpha support for expanding persistent volumes. We currently following volume types:
```

Comment:
>We currently following volume types:

In v1.9, the following types have volume expansion support:

Reply:
fixed